### ISSUE TITLE
Invalid certificate

### ISSUE THREAD

**Issue by nobu** [seattlerb/minitest#647]
```
$ ruby2.3 -rrubygems/package -e 'Gem::Package.new(ARGV[0], Gem::Security::LowSecurity).verify' minitest-5.9.0.gem 
/opt/local/lib/ruby2.3/2.3.0/rubygems/security/policy.rb:100:in `check_cert': certificate /CN=ryand-ruby/DC=zenspider/DC=com not valid after 2016-09-18 20:51:22 UTC (Gem::Security::Exception)
	from /opt/local/lib/ruby2.3/2.3.0/rubygems/security/policy.rb:246:in `verify'
	from /opt/local/lib/ruby2.3/2.3.0/rubygems/security/policy.rb:288:in `verify_signatures'
	from /opt/local/lib/ruby2.3/2.3.0/rubygems/package.rb:528:in `verify'
	from -e:1:in `<main>'
```

**Comment by nobu** [comment]
`gem install` fails too.

```
$ gem2.3 install -P LowSecurity minitest-5.9.0.gem 
ERROR:  While executing gem ... (Gem::Security::Exception)
    certificate /CN=ryand-ruby/DC=zenspider/DC=com not valid after 2016-09-18 20:51:22 UTC
```

**Comment by zenspider** [comment]
Sorry. I should have closed this a while ago. It should be addressed via 5.9.1 and 5.8.5 on 2016-01-21.
